### PR TITLE
Default Stub Triggers for NPSP Objects

### PR DESCRIPTION
**Pull request by force2b**
# Critical Changes

* Add all remaining Trigger Events to the TDTM_GeneralAccountingUnit trigger. This previously only supported the "Before Delete" event.
* Created stub Triggers for the following NPSP Custom Objects
    * Grant_Deadline__c
    * DataImport__c
    * DataImportBatch__c
* Created a stub Trigger for the OpportunityContactRole object

# Changes

# Issues Closed

# New Metadata

* New Apex Triggers:
    * TDTM_GrantDeadline
    * TDTM_DataImport
    * TDTM_DataImportBatch
    * TDTM_OpportunityContactRole

# Deleted Metadata

### CODE REVIEW

**Comment by force2b** [comment]
**lurch: attach W-035124

**Comment by LurchTheButler** [comment]
Tracking <a href="https://foundation.my.salesforce.com/a2x1E000001C92WQAS" target="_blank">W-035124</a>

File: src/triggers/TDTM_Batch.trigger [hunk 1]
```diff
@@ -0,0 +1,35 @@
+/*
```
Comment (ptewson-sfdo) [inline src/triggers/TDTM_Batch.trigger]: The Batch__c sObject, I believe, is mostly-entirely deprecated at this point as it serves the now sunsetted Batch Data Entry feature. Maybe we cover with a stub trigger anyway but just wanted to make sure we were clear that the Batch__c object is no longer really in service.

**Comment by rponti-sforg** [comment]
QE testing is complete for this PR.  Tests have passed.

For further testing details, see: 
https://salesforce.quip.com/XPXAAAEQ2pS
https://foundation.lightning.force.com/lightning/r/agf__ADM_Work__c/a2x1E000001C92WQAS/view

Merging PR and setting corresponding WI to Closed.

**Comment by sam-knox** [comment]
@force2b - Does this PR introduce new Trigger Handler records as well? Just thinking that we might need to update our Apex Class Descriptions and Default Trigger Handler Configuration articles.

**Comment by force2b** [comment]
@sam-knox No. These are stub triggers only. Customers can now implement their own trigger handlers on these objects without having to also implement their own trigger to work the trigger handler.

**Pull request merged**